src/benchmarklib/tpch/tpch_queries.cpp

```diff
@@ -235,10 +235,10 @@ const char* const tpch_query_6 =
  * ORDER BY supp_nation, cust_nation, l_year;
  *
  * Changes:
- *  1. dates are not supported
+ *  1. SQLite does not support dates and intervals
  *    a. use strings as data type for now
  *    b. pre-calculate date operation
- *  2. Extract is not supported
+ *  2. SQLite does not support extract
  *    a. Use SUBSTR instead (because our date columns are strings AND SQLite doesn't support EXTRACT)
```

Comment:
Bot yours, but I thinks it’s `substring` in TPC-H.

Reply:
[TPC-H 3.0](http://tpc.org/tpc_documents_current_versions/pdf/tpc-h_v3.0.0.pdf) seems to use `extract(...)` in Q7-Q9 (pp. 39ff)

Reply:
Yeah, correct. But also `substr/substring` in other places and I think they switched to `substring` years ago.
So one more thing we can get more compliant with.

Reply:
Ah okay, I think I got you wrong there 😅 
Yeah, [reference system](https://www.postgresql.org/docs/current/functions-string.html) seems to use substrings, too.